tests/PKSim.UI.Tests/PKSim.UI.Tests.csproj

```diff
@@ -48,7 +48,7 @@
 		<ItemGroup>
 			<WindowsFiles Include="$(TargetDir)x64/SQLite.Interop.dll" />
 		</ItemGroup>
-		<Copy SourceFiles="@(WindowsFiles);" DestinationFolder="$(TargetFolder)" DestinationFiles="@(WindowsFiles->Replace('x64/SQLite.Interop.dll', 'SQLite.Interop.dll'))" />
+		<Copy SourceFiles="@(WindowsFiles);" DestinationFolder="$(TargetFolder)" DestinationFiles="@(WindowsFiles-&gt;Replace('x64/SQLite.Interop.dll', 'SQLite.Interop.dll'))" />
```

Comment:
@rwmcintosh Any idea if these changes are legit? This was done automatically when I updated nuget packages